src/styles/index.scss

```diff
@@ -1,3 +1,20 @@
-body {
+@import './blocks/card';
+@import './blocks/stars';
+@import './variables';
```

Comment:
It's generally a good practice to import the `variables` file before other style blocks that use these variables. Consider moving the `@import './variables';` line to the top to ensure all variables are available for use in the imported styles.